plugin/gf/user.vim

```diff
@@ -50,10 +50,11 @@ call s:define_ui_key_mappings()
 
 
 
-command! -bang -bar -nargs=0 GfUserDefaultKeyMappings
-\ call s:cmd_GfUserDefaultKeyMappings(<bang>0)
-function! s:cmd_GfUserDefaultKeyMappings(banged_p)
-  let modifier = a:banged_p ? '' : '<unique>'
+command! -bang -nargs=* GfUserDefaultKeyMappings
```

Comment:
```suggestion
command! -bang -bar -nargs=* GfUserDefaultKeyMappings
```

`-bar` should not be dropped unless the command requires `|` as a part of its argument.